docs/reference/buildx_create.md

```diff
@@ -125,6 +125,8 @@ Passes additional driver-specific options. Details for each driver:
     - `image=IMAGE` - Sets the container image to be used for running buildkit.
     - `namespace=NS` - Sets the Kubernetes namespace. Defaults to the current namespace.
     - `replicas=N` - Sets the number of `Pod` replicas. Defaults to 1.
+    - `requests=CPU:MEMORY` - Sets the requests resources. It is allowed to ommit one of the resources - ex. `requests=:100Mi`, `requests=100m:`
```

Comment:
Can we introduce multiple flags like requests.cpu and requests.memory? Or requests_cpu.

Reply:
Well, its not a problem. I just wanted to avoid that many new flags.